docs/hld/projects/publish/high-level-design.md

```diff
@@ -0,0 +1,22 @@
+# High Level Design
+* According to the [design](https://www.figma.com/file/hq0BZNB9fzyFSbEUgQIHdK/Marxan-Visual_V02?node-id=4684%3A7130) the published project can have its own:
+    * name
+    * description
+    * tags
+    * creators
+    * scenario thumbnail
+* User can see frequency maps, but not features ([refs](https://vizzuality.atlassian.net/browse/MARXAN-864?focusedCommentId=13137))
+* We shouldn’t need to show protected areas for public projects in public view (non-authenticated users) ([refs](https://vizzuality.atlassian.net/browse/MARXAN-863?focusedCommentId=13138))
+
+# To be declared 
```

Comment:
Further things that may be an issue:

* what happens if someone adds another scenario to already published project? Is it also public?
* how thumbnail will be generated? (some job to run headless chrome and render tile?)
* where the thumbnail will be stored? (can we access it through cdn?)
* should we care about SEO of public project page?

Reply:
@Sikora00 we don't yet have any thumbnails. I suppose we should mention a spike needed (who/when/how captures, how we cache/invalidate...)

Reply:
yes, I'd say these questions should not block this document to be agreed upon.